src/actions/app.js

```diff
@@ -200,6 +212,18 @@ export const { applySort } = createActions({
   }),
 });
 
+export const { resetQuery } = createActions({
```

Comment:
If these new actions are a part of the new preview API, shouldn't they be declared under the `previewData` constant?

Reply:
Any of the "query" actions apply to all of the tables, so it wouldn't be appropriate to move this under previewData.